CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Revision history for plutarch
 
+# Unreleased
+
+- `TermCont`: Parametrize by result type; add `MonadFail` instance; etc.
+
+  Also, export from `Plutarch.TermCont`, and then from `Plutarch.Prelude` (TermCont is no longer exported by `Plutarch.Internal`).
```

Comment:
It's also exported from `Plutarch` now. I don't think the "(TermCont is no longer exported by `Plutarch.Internal`)" part is really needed. Users shouldn't care about restructures of internal modules.

Reply:
The internal stuff isn't necessary yeah. I don't think the other stuff needs to change though.